wagtail/contrib/search_promotions/models.py

```diff
@@ -37,8 +60,11 @@ def hits(self):
     @classmethod
     def garbage_collect(cls):
         """
-        Deletes all Query records that have no daily hits or editors picks
+        # Deletes all Query records that have no daily hits or editors picks
```

Comment:
The `#` in front of lines is not really necessary, we don't really do that in any other code.